debug/src/SimpleClient.hs

```diff
@@ -226,7 +229,7 @@ runOn (sStorage, certStore) flags port hostname
                 when (Verbose `elem` flags) $ printHandshakeInfo ctx
                 sendData ctx $ query
                 loopRecv out ctx
-                bye ctx
+                bye ctx `catch` \(SomeException _) -> return ()
```

Comment:
As this is a debugging tool I would still print the exception (and continue).
Not all errors are interesting but some may be.